rust/src/bwrap.rs

```diff
@@ -122,7 +138,7 @@ impl Drop for RoFilesMount {
             return;
         };
         // We need to unmount before letting the tempdir cleanup run.
-        let success = Command::new("fusermount")
+        let success = Command::new(get_fusermount_path().unwrap().to_string())
```

Comment:
We could factor this out into a helper that we run once at startup, that would help us avoid the unwrap() but I'm fine with this as is too

Reply:
let's see if this fixes CI with the spec patch and I'll do a follow up with the startup part? 

I guess it would go here https://github.com/coreos/rpm-ostree/blob/cd6fd88b39889b5ee3f06f9c8abd48a0813970f9/rust/src/daemon.rs#L25 ?